### PR TITLE
Never scroll when selecting a fact by clicking on it

### PR DESCRIPTION
**Pull request by paulwarren-wk**
#### Reason for change

Currently, when selecting a fact, we check to see if the fact is fully visible, and if not, scroll so that is is centered in the view.

The code doesn't take into account continuations, so if you click on a continuation, and the "head" fact isn't fully in view, we scroll away from what you clicked on.  This is also a problem even without continuations if you have a large text block that is larger than the viewport.

On reflection, when you select a fact by clicking on it in the document view, I don't think it's ever desirable to scroll that view.

#### Description of change

We now never scroll the view

#### Steps to Test

See steps described on #437 (note this isn't a fix for the main part of #437, just the fact that it scrolls away from what you click on).

Alternatively, scroll so that any fact is only partially visible.  Click on it, and notice that the view no longer scrolls to center it.

**review**:
@Workiva/xt
@paulwarren-wk

### CODE REVIEW

File: iXBRLViewerPlugin/viewer/src/js/viewer.js [hunk 1]
```diff
@@ -547,14 +547,15 @@ export class Viewer {
...
+     *
+     * byClick indicates that the element was clicked directly, and in this
+     * case we never scroll to make it more visible.
      */
-    selectElement(itemId, itemIdList) {
+    selectElement(itemId, itemIdList, byClick) {
         if (itemId !== null) {
-            this.onSelect.fire(itemId, itemIdList);
-        }
-        else {
```
Comment (lukepint-wk) [inline iXBRLViewerPlugin/viewer/src/js/viewer.js:555]: Can you explain the removal of the `else` here? Why do we want to call `this.onSelect.fire(null);` unconditionally?

Comment (derekgengenbacher-wf) [reply iXBRLViewerPlugin/viewer/src/js/viewer.js:555]: @paulwarren-wk

Comment (paulwarren-wk) [reply iXBRLViewerPlugin/viewer/src/js/viewer.js:555]: This was a mistake.  I somehow imagined a `return` inside the `if`.  Reverted in 13a36c3

**Review by brettkail-wk — APPROVED** [review]

**Review by lukepint-wk — APPROVED** [review]

**Comment by derekgengenbacher-wf** [comment]
QA +10 Unit and Skynet tests pass successfully. Builds successfully.

@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**